### ISSUE TITLE
Merging inspect and report

### ISSUE THREAD

**Issue by jacarrico** [assemblerflow/flowcraft#181]
Using flowcraft repeateadly I arrived to the conclusion that if the report can be only a flag on the inspect mode it would be better and would avoid to have 2 open terminal to get the same result. 
What are your opinions?

**Comment by ODiogoSilva** [comment]
That could work, but the two services should be kept separate. We could either create a new flowcraft mode (`flowcraft one-seeing-eye`) that would actually run the two services in parallel and return both links? Or add a flag to each `report` and `inspect` to track each one, but that seams less intuitive.

**Comment by jacarrico** [comment]
@ODiogoSilva I agree to keep it separate because the user might want the option not to send results data for a webservice. It returns the inspect link with the button and the popup saying reports are available

**Comment by cimendes** [comment]
Usually when you already a have a complete analysis you don't return to the insect of the pipeline. It's arguably much more useful to have both services running while a pipeline is still executing. An alternative to the new flowcraft mode is to add a flag to the inspect mode to also launch the report service, in watch mode. Something like `flowcraft inspect -m broadcast --report`). I'm not against the suggested new mode by any means.
